_build/templates/default/sass/dark-theme.scss

```diff
@@ -0,0 +1,552 @@
+@media (prefers-color-scheme: dark) {
+  body {
+    color: #b1b1b5 !important;
```

Comment:
For better maintainability you can use variables: https://sass-lang.com/documentation/variables
`#b1b1b5`: 42 matches.

Reply:
@mishantrop Thank you for message. I will try to continue working on the design theme. But this will not be with native support for a dark theme, but the one selected by the user in the system settings.

And of course everything will be in variables =).

Thanks!

Reply:
@mishantrop Thank you for message. I will try to continue working on the design theme. But this will not be with native support for a dark theme, but the one selected by the user in the system settings.

And of course everything will be in variables =).

Thanks!